.github/workflows/dockerhub.yml

```diff
@@ -43,12 +34,23 @@ jobs:
         uses: docker/metadata-action@9ec57ed1fcdbf14dcef7dfbe97b2010124a938b7
         with:
           images: telicent/elasticsearch
-      
-      - name: Build and push Docker image
-        uses: docker/build-push-action@3b5e8027fcad23fda98b2e3ac259d8d67585f671
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@master
```

Comment:
Do you want these pinned to appropriate sha1 hashes?

Reply:
Yes, maybe to keep with what was done on the other actons

Reply:
Now pinned versions appropriately